### PR TITLE
network: bump netlink receive buffer size to 128M

### PR DESCRIPTION
**Pull request by yuwata**
Prompted by #14417.

### CODE REVIEW

File: src/network/networkd-manager.c [hunk 1]
```diff
@@ -39,7 +39,7 @@
 #include "virt.h"
 
 /* use 8 MB for receive socket kernel queue. */
```
Comment (dnicolodi) [inline src/network/networkd-manager.c]: Comment needs to be updated too

Comment (yuwata) [reply src/network/networkd-manager.c]: Thanks. Updated.

File: units/systemd-networkd.socket [hunk 2]
```diff
@@ -15,7 +15,7 @@ DefaultDependencies=no
 Before=sockets.target
 
 [Socket]
-ReceiveBuffer=8M
+ReceiveBuffer=128M
```
Comment (haslersn) [inline units/systemd-networkd.socket:18]: Is this memory permanently reserved, or on demand?

Comment (poettering) [reply units/systemd-networkd.socket:18]: It's a limit, not more. i.e. when nothing is in the buffer no memory is allocated for it. And if there's little in the buffer only little is allocated.

**Pull request merged**